core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2677,11 +2564,19 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_NETWORK_KEEPALIVE_TIME_MS =
+      new Builder(Name.USER_NETWORK_KEEPALIVE_TIME_MS)
+          .setDefaultValue(Integer.MAX_VALUE)
```

Comment:
is this default correct?

Reply:
Yes both gRPC and Alluxio 1.x do not perform client side keep alive.